dev-requirements.txt

```diff
@@ -4,3 +4,4 @@ black==22.3.0
 isort==5.12.0
 pre-commit
 shellcheck-py==0.9.0.6
+reformat-gherkin
```

Comment:
Should we pin the dependency to a version?

Reply:
Yes, my bad. I think we should pin it to 3.0.1. Thanks for pointing that out.